iosApp/iosApp/Pages/TripDetails/TripDetailsStopView.swift

```diff
@@ -26,9 +26,29 @@ struct TripDetailsStopView: View {
                         Spacer()
                         UpcomingTripView(prediction: upcomingTripViewState, routeType: routeType)
                     }
+                    .accessibilityElement(children: .combine)
+                    .accessibilityAddTraits(.isHeader)
+                    .accessibilityHeading(.h2)
                 }
             )
             scrollRoutes
+                .accessibilityElement()
+                .accessibilityLabel(scrollRoutesAccessibilityLabel)
+        }
+    }
+
+    var scrollRoutesAccessibilityLabel: String {
+        if stop.routes.isEmpty {
+            return ""
+        } else if stop.routes.count == 1 {
+            return "Connection to \(stop.routes.first!.label) \(stop.routes.first!.type.typeText(isOnly: true))"
+        } else {
+            let lastConnection = stop.routes.last
+            return stop.routes.prefix(stop.routes.count - 1).map {
+                "\($0.label) \($0.type.typeText(isOnly: true))"
+            }
+            .joined(separator: ", ") +
+            "and \(lastConnection!.label) \(lastConnection!.type.typeText(isOnly: true))"
```

Comment:
Oh I missed this: I think we'll want a space before "and ...", otherwise it reads "bus and" like one word